Supporting Documents/BS_SD.adoc

```diff
@@ -657,6 +657,48 @@ The evaluator can pass this EA only if the evaluator confirms that:
 
 The evaluator shall report the summary of result of EA defined above, especially how the evaluator reaches the Pass/Fail judgement based on the Pass/Fail criteria.
 
+=== Trusted Path/Channels (FTP)
+==== EA for FTP_TRP.1
+===== Objective of the EA
+The path between the biometric capture sensor and the SEE where processing of the biometric characteristics will be performed per <<FPT_BDP_EXT.1>> or interactions to the main computer operating system occur (such as providing the results of a match), must protect the captured data from being tampered with or duplicated in transit. The evaluator shall confirm that this data path is protected from access by untrusted components (generally defined as anything in the main computer operating system). 
```

Comment:
Should "match" be "comparison"? (See above change (Line 868)

Reply:
Yes, per ISO/IEC 2382-37. The "result of a match" could be a "comparison decision."